### ISSUE TITLE
[Suggestion] Improve drasticly ship fighting with this

### ISSUE THREAD

**Issue by falco708** [tsunamayo/Starship-EVO#3137]
Right now ship fighting in this game is just 99% of other spaceship fighting game: dogfight airplane  
I think this is totally wrong, you can for very small and agile ship use this gameplay and make it feel fun but the bigger your ship will be the more wrong it will feel.

Of course you could just go for the "have 20 player in one ship everyone at one turret and you're done" but it will almost never happen.

Then according to me the perfect fight system compatible with solo player is the following:
While piloting a ship in 3rd person, stop the ship from following pilot's mouse cursor and make all turrets follow his cursor.
The goal is to make the pilot play with turret and shoot with without moving his ship if he don't want to (or keep moving it with WASD while aiming and shooting).

The big difference is that you will not need anymore to turn, turn, turn, turn, shoot, turn, turn, turn, turn, turn, shoot,... 

I'm not sure if you can imagine the playstyle it will lead but it is honestly the best thing possible for spaceship. You will be able to shoot with all sort of different weapons (missile, beam, mines, guns, pulsar, anything you can imagine) while always keep an eye on the enemy and the battle. You will be able make way more strategic move and choice during the battle other than dogfighting forever.

Now get this with better 3rd person view (ability to zoom in or zoom out with easily for example), better HUD, a bit better enemy AI and you're good for big spaceship battles.

You can give a look at Fractured Space which did this fight mechanic very well (but the game did absolutely nothing else unfortunally that's why nobody played it)
Have a look on youtube or try [this link ](https://youtu.be/BxYcOG7yEE8?t=86) you can see the player can clearly spot enemy ship from dead angle, shoot on it without turning around for hours, watch the map and other ship clearly and decide where to move smartly, have the time to set up ship ability like cloaking and other stuff, have fun using a lot of different weapons and watch all of them be fired which is a big part of the fun.
This fighting mechanic honestly work so well it's insane almost no space fighter game used it.

What do you think ?

**Comment by tsunamayo** [comment]
Hi, so I must understand that you are not using your turret in auto-mode, correct? There was a suggestion about this, to have WASD become turn key instead of strafe key when you are in free look mode. I was thinking it added some complexity but it is still a valid solution to this problem.
Personally I just use turret in auto-mode, and I will implement crew at one point.

**Comment by falco708** [comment]
yeah using WASD into strafe key in free look would be a nice start option to work on
I was speaking about the manual use of turret to be able to shoot anywhere without rotating the ship
I still think in game design that it would be way more fun in every way to shoot manually, control all turret manually, see our weapons shooting great missiles, beams, and stuffs, and watch the enemy health getting lower while moving our ship.
Again dogfight is ok for small ship but a pain for big one.
A crew is also something interesting but too complex and it take away a part of the fun for me if everything shoot in auto-mode.

**Comment by matejkar** [comment]
I agree with falco's idea, as i had a similar one at https://github.com/tsunamayo/Starship-EVO/issues/3090
